dashboard/app/views/maker/setup.html.haml

```diff
@@ -1,5 +1,4 @@
-%script{src: minifiable_asset_path('js/makerlab.js')}
```

Comment:
That's a good question.  Right now my answer is "I don't know."  It looks like we stopped generating this file at some point (it 404s on my local build) but it still exists on prod, and my changes here did not stop building it - we must be referring to an old version on production?

I will track down where this got removed from our build process.